1-js/11-async/07-microtask-queue/article.md

```diff
@@ -93,20 +93,20 @@ setTimeout(() => promise.catch(err => alert('caught')), 1000);
 window.addEventListener('unhandledrejection', event => alert(event.reason));
 ```
 
-Now, if you run it, we'll see `Promise Failed!` first and then `caught`. 
+Now if we run it, we'll see `Promise Failed!` first and then `caught`.
```

Comment:
I would prefer the comma be left as was, but changing "you" to "we" makes sense.